src/components/eventsCaraousal/Header.jsx

```diff
@@ -0,0 +1,17 @@
+import { Heading1, Heading2 } from '../shared/Typography/Headings';
+
+export const DescriptionHeader = ({ label }) => {
+  return (
+    <>
```

Comment:
instead of defining two new components, directly import Heading1 and heading2 in main.jsx
if there are no additional styling changes